### ISSUE TITLE
Enable to use keyboard shortcut and/or custom keyboard shortcut

### ISSUE THREAD

**Issue by kechan** [doccano/doccano#1808]
Feature description
---------
Labelling is a very manual and laborious task. It maybe great to have multiple means of input to reduce dragging and click of mouse, as well as diversifying hand/finger movement to reduce chance of repetitive injury. 

Default custom keyboard shortcut to popular function, and then later, maybe add ability to customize.

**Comment by Mukilan-Krishnakumar** [comment]
Yeah, I am really struggling right now with labeling 1000 text files. It would be amazing to have this feature.
